modules/cert-manager-acme-http01.adoc

```diff
@@ -0,0 +1,119 @@
++
+.Example `acme-cluster-issuer.yaml` file
+[source,yaml]
+----
+apiVersion: cert-manager.io/v1
+kind: ClusterIssuer
+metadata:
+  name: letsencrypt-staging                                        <1>
+spec:
+  acme:
+    preferredChain: ""
+    privateKeySecretRef:
+      name: <secret_for_private_key>                               <2>
+    server: https://acme-staging-v02.api.letsencrypt.org/directory <3>
+    solvers:
+      - http01:
+          ingress:
+            class: openshift-default                               <4>
+----
+<1> Provide a name for the cluster issuer.
+<2> Replace `<secret_private_key>` with the name of secret to store the ACME account private key in.
+<3> Specify the URL to access the ACME server's `directory` endpoint. This example uses the _Let's Encrypt_ staging environment.
+<4> Specify the ingress class. This must be `openshift-default`.
+
+.. Create the `ClusterIssuer` object by running the following command:
++
+[source,terminal]
+----
+$ oc create -f acme-cluster-issuer.yaml
+----
+
+. Create an ingress to expose the service of the user workload.
+
+.. Create a YAML file that defines a `Namespace` object:
++
+.Example `namespace.yaml` file
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: my-ingress-namespace <1>
+----
+<1> Specify the namespace for the ingress.
+
+.. Create the `Namespace` object by running the following command:
++
+[source,terminal]
+----
+$ oc create -f namespace.yaml
+----
+
+.. Create a YAML file that defines the `Ingress` object:
++
+.Example `ingress.yaml` file
+[source,yaml]
+----
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: sample-ingress                                           <1>
+  namespace: my-ingress-namespace                                <2>
+  annotations:
+    cert-manager.io/cluster-issuer: letsencrypt-staging          <3>
+    acme.cert-manager.io/http01-ingress-class: openshift-default <4>
+spec:
+  ingressClassName: openshift-default                            <5>
+  tls:
+   - hosts:
+       - <ingress_domain_name>                                   <6>
+     secretName: sample-tls                                      <7>
+  rules:
+  - host: <ingress_domain_name>                                  <8>
+    http:
+      paths:
+      - path: /
+        pathType: Prefix
+        backend:
+          service:
+            name: sample-workload                                <9>
+            port:
+              number: 80
+----
+<1> Specify the name of the ingress.
+<2> Specify the namespace that you created for the ingress.
+<3> Specify the cluster issuer that you created.
+<4> Specify the ingress class from the cluster issuer.
+<5> Specify the ingress class from the cluster issuer. (TODO: Update this per PR comment)
+<6> Replace `<ingress_domain_name>` with the ingress domain name. You can retrieve this name by running the following command: `oc get ingresses.config/cluster -o jsonpath={.spec.domain}`.
```

Comment:
Correction here, 

```suggestion
<6> Replace `<host_name>` with the [Subject Alternative Name](https://en.wikipedia.org/wiki/Subject_Alternative_Name) to be associated with the certificate (aka the secret). This is used to add DNS names to the certificate. 
```

Reply:
@thejasn The line for callout <6> doesn't have a <host_name> variable. It currently reads like this:

```
    - sample.<ingress_domain_name>                               <6>
```

Are you saying that, with this description change, we should update that to the following?


```
    - sample.<host_name>                               <6>
```